test/test_rhat.py

```diff
@@ -75,40 +88,47 @@ def test_split_rhat():
         split_rhat([[1, -2, 3, 4, 5, 6], [7, 8, 9, 12]]),
     )
 
-def split_rhat_throws(chains):
+
+def split_rhat_throws(chains: List[List[float]]) -> None:
     with pt.raises(ValueError):
         split_rhat(chains)
-        
-def test_split_rhat_at_least_one_chain():
+
+
+def test_split_rhat_at_least_one_chain() -> None:
     split_rhat_throws([])
 
-def test_split_rhat_at_least_four_elements_per_chain():
+
+def test_split_rhat_at_least_four_elements_per_chain() -> None:
     split_rhat_throws([[1, 2, 3]])
     split_rhat_throws([[1, 2, 3, 4], [1, 2, 3]])
     split_rhat_throws([[1, 2, 3], [1, 2, 3, 4]])
 
-def rank_chains_equal(ranks, chains):
+
+def rank_chains_equal(ranks: List[List[float]], chains: List[List[float]]) -> None:
     np.testing.assert_equal(ranks, rank_chains(chains))
-    
-def test_rank_chains():
+
+
+def test_rank_chains() -> None:
     rank_chains_equal([], [])
     rank_chains_equal([[1]], [[2.3]])
     rank_chains_equal([[1, 2]], [[2.3, 4.9]])
     rank_chains_equal([[2, 3, 1]], [[3.9, 5.2, 2.1]])
     rank_chains_equal([[2], [1]], [[4.2], [1.9]])
     rank_chains_equal([[2, 3], [1, 4]], [[4.2, 5.7], [1.9, 12.2]])
-    rank_chains_equal(
-        [[2, 3], [5, 4], [1, 6]],
-        [[4.2, 5.7], [7.2, 6.1], [-12.9, 107]]
-    )
+    rank_chains_equal([[2, 3], [5, 4], [1, 6]], [[4.2, 5.7], [7.2, 6.1], [-12.9, 107]])
```

Comment:
I'm tempted to overrule `black` here--the multi-line formatting makes this more legible, even if it does technically fit on one line.